test/unit/org/apache/cassandra/cql3/validation/operations/SelectMultiColumnRelationTest.java

```diff
@@ -52,21 +52,21 @@ public void testMultiClusteringInvalidQueries() throws Throwable
         createTable("CREATE TABLE %s (a int, b int, c int, d int, PRIMARY KEY (a, b, c, d))");
 
         assertInvalidSyntax("SELECT * FROM %s WHERE a = 0 AND (b, c) > ()");
-        assertInvalidMessage("Expected 2 elements in value for tuple (b,c), but got 3: (?, ?, ?)",
+        assertInvalidMessage("Expected 2 elements in value for tuple (b, c), but got 3: (?, ?, ?)",
                              "SELECT * FROM %s WHERE a = 0 AND (b, c) > (?, ?, ?)", 1, 2, 3);
-        assertInvalidMessage("Invalid null value in condition for column c",
+        assertInvalidMessage("Invalid null value for c in tuple (b, c)",
                              "SELECT * FROM %s WHERE a = 0 AND (b, c) > (?, ?)", 1, null);
 
         // Wrong order of columns
-        assertInvalidMessage("Clustering columns must appear in the PRIMARY KEY order in multi-column relations: (d, c, b) = (?, ?, ?)",
+        assertInvalidMessage("Clustering columns must appear in the PRIMARY KEY order in multi-column relations: (d, c, b)",
                              "SELECT * FROM %s WHERE a = 0 AND (d, c, b) = (?, ?, ?)", 0, 0, 0);
-        assertInvalidMessage("Clustering columns must appear in the PRIMARY KEY order in multi-column relations: (d, c, b) > (?, ?, ?)",
+        assertInvalidMessage("Clustering columns must appear in the PRIMARY KEY order in multi-column relations: (d, c, b)",
```

Comment:
Where did > (?, ?, ?) fell off? Seems useful info for the user

Reply:
I moved the check to a place where the remaining information is not available. At the same time the remaining information was for me more confusing that helping. The issue is with the columns not the operator or values.